src/components/Overview.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import { RouteComponentProps } from 'react-router-dom';
+import auth from '../auth/auth';
+
+interface OverviewProps extends RouteComponentProps<{name:string}> {
+
+}
+
+function Overview(props:OverviewProps) {
+  return (
+    <div>
+      This is the overview component
+      <button
+        type="button"
+        onClick={() => {
```

Comment:
it might be better to have this as a separately declared function in case we have a lot going on in this component